src/lint/practice_exercises.nim

```diff
@@ -25,12 +17,9 @@ proc checkFiles(data: JsonNode, context, path: string): bool =
 proc isValidPracticeExerciseConfig(data: JsonNode, path: string): bool =
   if isObject(data, "", path):
     result = true
-    # Temporarily disable the checking of authors as we'll be doing bulk PRs
-    # to pre-populate this field for all tracks
-    # if not hasArrayOf(data, "authors", path, isValidAuthorOrContributor):
-    #   result = false
-    if not hasArrayOf(data, "contributors", path, isValidAuthorOrContributor,
-                      isRequired = false):
+    if not hasArrayOfStrings(data, "", "authors", path):
```

Comment:
We're also making this field optional (for now). See https://github.com/exercism/docs/pull/88

```suggestion
    if not hasArrayOfStrings(data, "", "authors", path, isRequired = false):
```